common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1776,6 +1781,22 @@ object CelebornConf extends Logging {
       .timeConf(TimeUnit.SECONDS)
       .createWithDefaultString("5s")
 
+  val HA_MASTER_RATIS_CLIENT_RPC_TIMEOUT: ConfigEntry[Long] =
+    buildConf("celeborn.master.ha.ratis.raft.client.rpc.timeout")
```

Comment:
Could this add method to parse the config of Ratis itself, which could not alway follow the Ratis config change?

Reply:
Yes, there is a way to allow users who are familiar with Ratis to pass all their own Ratis configs to Celeborn.